README.md

```diff
@@ -41,13 +51,36 @@ Then, `git clone` this repository into your `$GOPATH` and `go build -o vault-plu
 
 The Vault plugin system is documented on the [Vault documentation site](https://www.vaultproject.io/docs/internals/plugins.html).
 
-You will need to define a plugin directory using the `plugin_directory` configuration directive, then place the `vault-plugin-database-oracle` executable generated above in the directory.
+You will need to define a plugin directory using the `plugin_directory` configuration directive, then place the
+`vault-plugin-database-oracle` executable generated above in the directory.
 
-Register the plugin using
+Sample commands for registering and starting to use the plugin:
 
 ```
-vault write sys/plugins/catalog/database/vault-plugin-database-oracle \
-    sha_256=<expected SHA256 Hex value of the plugin binary> \
+$ shasum -a 256 vault-plugin-database-oracle > /tmp/oracle-plugin.sha256
+
+$ vault write sys/plugins/catalog/database/vault-plugin-database-oracle \
+    sha_256=$(cat /tmp/oracle-plugin.sha256) \
```

Comment:
nit: `sha256` is the current version of the parameter. `sha_256` is deprecated

Reply:
:+1: